processing/src/main/java/io/druid/query/aggregation/SimpleDoubleAggregatorFactory.java

```diff
@@ -34,10 +35,12 @@
 
 public abstract class SimpleDoubleAggregatorFactory extends AggregatorFactory
 {
+  public final static String DOUBLE_STORAGE_TYPE_PROPERTY = "druid.indexing.doubleStorage";
```

Comment:
Hm, I think `io.druid.segment.column.Column` would be a better place for this property and the `storeDoubleAsFloat()` method